apps/src/standaloneVideo/VideoJS.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useRef} from 'react';
+import videojs, {VideoJsPlayer, VideoJsPlayerOptions} from 'video.js';
+import 'video.js/dist/video-js.css';
```

Comment:
How much is in this css file? My understanding is that direct imports like this can affect others styling on the page.

Reply:
Great question.  It can be perused (though not directly linked) starting [here](https://www.npmjs.com/package/video.js/v/7.6.6?activeTab=code).  It's 45.3 kB, 1664 lines, and does seem scoped to its own components.

Reply:
Yeah, interesting that the docs suggest importing the css file directly from `dist`. +1 to Ben's comment that, though very unlikely, this could cause some issues if any of the style rules collide with ours.

Would it make sense to style the player ourselves (using our colors/fonts)?